firestore.rules

```diff
@@ -15,5 +15,20 @@ service cloud.firestore {
     match /{document=**} {
       allow read, write: if request.time < timestamp.date(2024, 10, 25);
     }
+
+    match /applicantUsers/{userId} {
+      allow write: request.resource.data.name != null &&
```

Comment:
These rules aren't compiling when I run them using the emulator. The applicantUsers rule is definitely missing an "if" but I'm also getting an error on "size". Let me know if you guys need any help testing with the emulator